### DISCUSSION TITLE
Optimierungen Inventarmaske

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#8] [Shipped - Live on production]
1. Warum sind diese Felder mit * markiert? Sind weder rechtlich verpflichtend, noch Pflichtfelder

![Bildschirmfoto 2022-05-31 um 15 01 43](https://user-images.githubusercontent.com/80519295/171607775-66ebafb1-7953-44a8-9a91-2bb13381b5f6.png)

2. Position Servicepartner. Beim Inventar anlegen in Step 1 und nach Abschluss auf Tab "Ereignisse"

<img width="1158" alt="Bildschirmfoto 2022-06-02 um 12 13 50" src="https://user-images.githubusercontent.com/80519295/171608358-73c428b4-a1a0-4c38-8620-dc4a20b40edd.png">
<img width="1158" alt="Bildschirmfoto 2022-06-02 um 12 14 25" src="https://user-images.githubusercontent.com/80519295/171608466-a26c5d3d-0c04-448e-88b2-687b07334342.png">

3. Feld Gerätezustand

Ist schon sehr groß und warum haben wir das wirklich separat zum Bemerkungsfeld eingeführt?

 ![Bildschirmfoto 2022-05-31 um 15 09 51](https://user-images.githubusercontent.com/80519295/171608705-2709d141-3b93-457f-ac33-28766b2ff9d8.png)

**Comment by YvesR** [comment]
1. Bug
2. Wir bereits geändert, siehe https://github.com/Samedis-care/samedis-care-issues/issues/581
3. War Kundenwunsch und ist in der App mit drin für Inventarisierung.
